### PR TITLE
 SubmitFeed & GetFeedSubmissionResult add MWSAuthToken parameter

### PR DESCRIPTION
**Pull request by ghost**
You can use MWSAuthToken parameter at SubmitFeed & GetFeedSubmissionResult.

https://docs.developer.amazonservices.com/en_US/feeds/Feeds_SubmitFeed.html

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,15 +1,18 @@
 {
-    "name": "tilleuls/amazon-mws",
```
Comment (pborreli) [inline composer.json:2]: 😄

Comment (magazord-plataforma) [reply composer.json:2]: sorry, the file composer.json is not part of the change, but I could not delete it from the pull.

I believed that the pull would only pick up changes until the pull creation.

I will reverse again.

Comment (pborreli) [reply composer.json:2]: no problem :)

**Review by livehybrid — COMMENTED** [review]
+1 on this :)

**Comment by samjross** [comment]
I'm pretty sure this is the only issue actually stopping me from being able to use this package. There needs to be support for MWSAuthToken

File: src/MarketplaceWebService/Client.php [hunk 2]
```diff
@@ -1173,7 +1173,7 @@ private function getDefaultCurlOptions()
         return array(
             CURLOPT_POST => true,
             CURLOPT_USERAGENT => $this->config['UserAgent'],
-            CURLOPT_VERBOSE => true,
+            // CURLOPT_VERBOSE => true,
```
Comment (Deuchnord) [inline src/MarketplaceWebService/Client.php:1176]: If the line is not required anymore, you can delete it

**Pull request closed without merging**